app/components/windows/EditStreamInfo.vue.ts

```diff
@@ -15,25 +15,23 @@ import { NavigationService } from 'services/navigation';
 import { CustomizationService } from 'services/customization';
 import { Multiselect } from 'vue-multiselect';
 import { $t } from 'services/i18n';
+import FormInput from 'components/shared/inputs/FormInput.vue';
+import { IStreamlabsFacebookPage, IStreamlabsFacebookPages } from 'services/platforms/facebook';
 import {
   VideoEncodingOptimizationService,
-  IEncoderPreset,
+  IEncoderProfile,
 } from 'services/video-encoding-optimizations';
-import { IStreamlabsFacebookPage, IStreamlabsFacebookPages } from 'services/platforms/facebook';
 import { shell } from 'electron';
+import VFormGroup from 'components/shared/inputs/VFormGroup.vue';
 import { IListOption } from '../shared/inputs';
 
-interface IMultiSelectProfiles {
-  value: IEncoderPreset;
-  description: string;
-  longDescription: string;
-}
-
 @Component({
   components: {
     ModalLayout,
-    BoolInput,
+    FormInput,
```

Comment:
not sure why this is here

Reply:
Looks like something that I left here after the merge conflict resolving